### DISCUSSION TITLE
Wrong Date on Webinar Info?

### DISCUSSION THREAD

**Discussion by riziles** [executablebooks/meta#971] [Q&A] [answered]
@rowanc1 or anyone else who sees this message:

The webinar announcement here (https://executablebooks.org/en/latest/blog/2023-03-07-call-for-myst-templates/) says the webinar is on 15th March 2022. Should it be 2023?

**Comment by rowanc1** [comment, marked as answer]
It's 2023!?

(Will fix the post, thank you!!!)

**Reply by rowanc1** [reply]
Fix has been deployed -- thanks @mmcky for the merge. :)
